### DISCUSSION TITLE
Material UI not working without reload if any error handle

### DISCUSSION THREAD

**Discussion by ShourovRoy** [remix-run/remix#2236] [Q&A]
Remix is not working properly with Material Ui. Its working with reloads. But when 404 page is coming and then returning back by back key then whole material design is removed. Please help

**Comment by dan-cooke** [comment]
I have been using MUI perfectly fine with remix for the last month - so it should work...

If you have a reproduction repo someone here can maybe help!

**Reply by ShourovRoy** [reply]
> I have been using MUI perfectly fine with remix for the last month - so it should work...
> 
> If you have a reproduction repo someone here can maybe help!

Mui is working fine but at the case of back key to getback to the previous page It became style less completely until a reload

**Comment by tombohub** [comment]
@dan-cooke  do you have template for MUI with Remix?
